showcase/src/elm/UI/Container.elm

```diff
@@ -228,32 +257,40 @@ iconContainer { icon, tooltipText, event, extraStyles } =
                 , opacityTransition
                 ]
 
-        childNode =
+        viewIcon extraStyles icon enabled =
             icon
-                |> Icons.withStyles (extraStyles ++ commonIconStyles)
+                |> Icons.withStyles (extraStyles ++ commonIconStyles enabled)
                 |> Icons.toHtml
                 |> fromUnstyled
-
-        bareIconContainer =
-            case event of
-                Left msg ->
-                    span
-                        [ baseAttributes
-                        , onClick msg
-                        ]
-                        [ childNode ]
-
-                Right ellieAppUrl ->
-                    Styled.a
-                        [ baseAttributes
-                        , A.target "_blank"
-                        , href ellieAppUrl
-                        ]
-                        [ childNode ]
     in
-    tooltip tooltipText (toUnstyled bareIconContainer)
-        |> Tooltip.toHtml
-        |> fromUnstyled
+    case options of
+        Disabled { icon, extraStyles } ->
+            span
+                [ baseAttributes ]
+                [ viewIcon extraStyles icon False ]
```

Comment:
My 2 cents:

Try to avoid having functions that take Boolean arguments. It's hard to infer what the meaning of the boolean is without having to go read the function definition.

Often what I do is create a binary algebraic data type such as `type DoAThing = Yes | No`, but I think in this case it's overkill / not worth the effort.

However, an improvement that I do recommend in this case is to make the Boolean argument the first argument for `viewIcon`. 

```elm
        viewIcon enabled extraStyles icon =
```

Then you can make functions out of this that are a lot more obvious / intuitive:

```elm
viewDisabledIcon = viewIcon False
viewEnabledIcon = viewIcon True

-- ...

[ viewDisabledIcon extraStyles icon ]
```

🎊 yay currying 🎊